integration/simulation/simulation_ganache_in_mem_test.go

```diff
@@ -35,7 +35,7 @@ func TestGanacheMemObscuroEthMonteCarloSimulation(t *testing.T) {
 	setupTestLog()
 
 	// private key is prefunded and used to issue txs - used here to deploy contract ahead of node initialization
-	tmpWallet := wallet.NewInMemoryWallet("5dbbff1b5ff19f1ad6ea656433be35f6846e890b3f3ec6ef2b2e2137a8cab4ae")
+	tmpWallet := wallet.NewInMemoryWallet("1e1e76d5c0ea1382b6acf76e873977fd223c7fa2a6dc57db2b94e93eb303ba85")
```

Comment:
out of curiosity, why is this changing?

Reply:
It's using a different PK than the ones used in tests. So I don't have to manage the nonce if the PK is reused.